doc/install/main.rst

```diff
@@ -0,0 +1,167 @@
+Manage Qserv deployment
+-----------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+For all setups
+^^^^^^^^^^^^^^
+
+-  Access to a Kubernetes v1.14.2+ cluster via a valid ``KUBECONFIG`` file.
+-  Dynamic volume provisionning need to be available on the Kubernetes cluster (for example `kind <https://kind.sigs.k8s.io/>`__ for or
+   GKE).
+
+For a development workstation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  Ubuntu LTS is recommended
+-  8 cores, 16 GB RAM, 30GB for the partition hosting docker entities
+   (images, volumes, containers, etc). Use ``df`` command as below to
+   find its size.
+
+   .. code:: bash
+
+       sudo df –sh /var/lib/docker # or /var/snap/docker/common/var-lib-docker/
+
+-  Internet access without proxy
+-  ``sudo`` access
+-  Install dependencies below:
+
+   .. code:: bash
+
+       sudo apt-get install curl docker.io git vim
+
+-  Add current user to docker group and restart gnome session
+
+   .. code:: bash
+
+       sudo usermod -a -G docker <USER>
+
+-  Install Kubernetes locally using this `simple k8s install script <https://github.com/k8s-school/kind-travis-ci>`__, based on
+   `kind <https://kind.sigs.k8s.io/>`__.
+
+
+Deploy qserv-operator
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: sh
+
+    # Deploy qserv-operator in current namespace
+    curl -fsSL https://raw.githubusercontent.com/lsst/qserv-operator/master/deploy/qserv.sh | bash -s --install-kubedb
+
+The ``--install-kubedb`` option enable ``qserv-operator`` to set-up a
+Redis cluster for managing its secondary index (OnjectID,ChunkId). It
+can be skipped for a regular Qserv installation.
+
+Deploy a qserv instance
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Deployments below are recommended for development purpose, or continuous
+integration. Qserv install customization is handled with
+`Kustomize <https://github.com/kubernetes-sigs/kustomize>`__, which is a
+template engine allowing to customize kubernetes Yaml files.
+``Kustomize`` is integrated with ``kubectl`` (``-k`` option).
+
+with default settings
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: sh
+
+    # Install a qserv instance with default settings inside default namespace
+    kubectl apply -k https://github.com/lsst/qserv-operator/overlays/dev --namespace='default'
+
+with a Redis cluster
+^^^^^^^^^^^^^^^^^^^^
+
+.. code:: sh
+
+    # Install a qserv instance plus a Redis cluster inside default namespace
+    # This overlay is used on Travis-CI, o, top of kind
+    kubectl apply -k https://github.com/lsst/qserv-operator/ci-redis --namespace='default'
+
+Undeploy a qserv instance
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First list all Qserv instances running in a given namespace
+
+.. code:: sh
+
+    kubectl get qserv -n "<namespace>"
+
+It will output something like:
+
+::
```

Comment:
The `::` syntax is a short cut for adding a colon to the end of the previous paragraph and make the next paragraph a code block. Right now this is adding an extra colon. I'd delete this and put a double colon at the end of `like`, above.